src/rest/SearchEndpointWithDefaultCallOptions.ts

```diff
@@ -139,6 +139,10 @@ export class SearchEndpointWithDefaultCallOptions implements ISearchEndpoint {
     return this.endpoint.logError(sentryLog);
   }
 
+  public exchangeAuthenticationProviderHandshakeTokenForAccessToken(token: string): Promise<string> {
```

Comment:
That's a mouthful ;) 

Suggestion:
```suggestion
  public exchangeAuthenticationProviderToken(token: string): Promise<string> {
```